docs/2022/index.md

```diff
@@ -0,0 +1,14 @@
+---
+sidebar_position: 1
+title: GSoC'22 @ FOSSology
+slug: /2022
+---
+
+<!--
+SPDX-License-Identifier: CC-BY-SA-4.0
+
+SPDX-FileCopyrightText: 2021 Gaurav Mishra <mishra.gaurav@siemens.com>
+SPDX-FileCopyrightText: 2021 Siemens AG
+-->
+
+2022 is the 5th year FOSSology participating in Google Summer of Code.
```

Comment:
So far FOSSology has not been announced as an org in GSoC, so we would like to rephrase the text as "FOSSology is looking forward to particiapting in Google Summer of Code in 2022."

Also, can you please provide a link to `docs/2022/GSoC-projects.md` with a title like "Idea page for GSoC 2022"?